content/metadata.yaml

```diff
@@ -1,680 +1,656 @@
-      - NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003
-  - github: jaclyn-taroni
-    name: Jaclyn N. Taroni*
-    initials: JNT
-    orcid: 0000-0003-4734-4508
-    twitter: jaclyn_taroni
-    email: jaclyn.taroni@ccdatalab.org
-    affiliations:
-      - Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Bala Cynwyd, PA, USA
-    funders:
-       - Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
-    contributions:
-      - Methodology
-      - Software
-      - Validation
-      - Formal analysis
-      - Investigation
-      - Data curation
-      - Writing - Review and editing
-      - Visualization
-      - Supervision
-      - Project administration
-  - name: Children's Brain Tumor Network
-    contributions:
-      - Conceptualization
-  - name: Pacific Pediatric Neuro-Oncology Consortium
-    contributions:
-      - Conceptualization
+- github: jashapiro
+  name: Joshua A. Shapiro
+  initials: JAS
+  orcid: 0000-0002-6224-0347
+  twitter: jashapiro
+  email: josh.shapiro@ccdatalab.org
+  contributions:
+  - Methodology
+  - Software
+  - Validation
+  - Formal analysis
+  - Investigation
+  - Writing - Original draft
+  - Writing - Review and editing
+  - Visualization
+  - Supervision
+  affiliations: Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Bala
+    Cynwyd, PA, USA
+  funders: Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
+- github: kgaonkar6
+  name: Krutika S. Gaonkar
+  initials: KSG
+  orcid: 0000-0003-0838-2405
+  twitter: aggokittu
+  email: ksgaonkar@outlook.com
+  contributions:
+  - Data curation
+  - Formal Analysis
+  - Investigation
+  - Methodology
+  - Software
+  - Writing – original draft
+  affiliations:
+  - Center for Data-Driven Discovery, Children's Hospital of Philadelphia
+  - Division of Neurosurgery, Children's Hospital of Philadelphia
+  - Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia
+- github: cansavvy
+  name: Candace L. Savonen
+  initials: CLS
+  orcid: 0000-0001-6331-7070
+  twitter: cansavvy
+  email: candace.savonen@ccdatalab.org
+  contributions:
+  - Methodology
+  - Software
+  - Validation
+  - Formal analysis
+  - Investigation
+  - Writing - Original draft
+  - Visualization
+  affiliations: Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Bala
+    Cynwyd, PA, USA
+  funders: Alex's Lemonade Stand Foundation Childhood Cancer Data Lab (CCDL)
+- github: sjspielman
+  name: Stephanie J. Spielman
+  initials: SJS
+  orcid: 0000-0002-9090-4788
+  twitter: stephspiel
+  email: stephanie.spielman@gmail.com
+  contributions:
+  - Validation
+  - Formal analysis
+  - Writing - Review and editing
+  - Visualization
+  - Supervision
+  affiliations:
+  - Childhood Cancer Data Lab, Alex's Lemonade Stand Foundation, Bala Cynwyd, PA,
+    USA<sup>☯</sup>
+  - Rowan University, Glassboro, NJ, USA
+  symbol_str: ☯
```

Comment:
I just want to note that this symbol renders as an emoji ☯️, which we probably don't want. Substitute † or ‡?